DexcomCGMKitUI/ViewModels/NotificationDetailsViewModel.swift

```diff
@@ -37,13 +37,26 @@ class NotificationDetailsViewModel: NotificationDetailsViewModelProtocol {
         
     @Published var threshold: Double = 0 {
         didSet {
-            configuration.threshold = HKQuantity(unit: configurationUnit, doubleValue: threshold)
+            var thresholdTemp = threshold
```

Comment:
[nit] Consider invariant instead of a `var` here:
```suggestion
            let thresholdConfig = configuration.type == .signalLoss && threshold != 0 ? threshold - TimeInterval.seconds(30) : threshold
```
Also, maybe it is just me, but I read "thresholdTemp" as "threshold temperature".

Reply:
Renamed to `tempThreshold`